### DISCUSSION TITLE
sh install-gui.sh cannot continue to install..

### DISCUSSION THREAD

**Discussion by furqanarda** [Chia-Network/chia-blockchain#6535] [Support]
Hello,
When i try to install Chia on Ubuntu, i get this warn message:

This requires the chia python virtual environment.
Execute '. ./activate' before running.

What do i have to do?

Thanks..

**Comment by loppefaaret** [comment]
you have to execute the command `. ./activate` as the error message says, and the install instructions also notes

**Comment by BrandtH22** [comment]
@furqanarda , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
